src/algebra/regular/mul_embedding.lean

```diff
@@ -0,0 +1,42 @@
+/-
```

Comment:
Since this file doesn't once mention `regular`, perhaps it belongs in `algebra.hom.embedding` to match `algebra.hom.equiv`?

Reply:
Indeed! Also the imports needed to shrink.